create-react-app/package.json

```
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "dev": "react-scripts start --port $PORT",
```

Where did you find this? We don't support this argument -- `$PORT` is already read and handled.
```suggestion
    "dev": "react-scripts start",
```